core/templates/pages/exploration-player-page/templates/information-card-modal.directive.html

```diff
@@ -108,4 +108,21 @@ <h2 ng-style="titleWrapper()" class="oppia-info-card-title"><[explorationTitle]>
     cursor: default;
     margin-right: 5px;
   }
+  .oppia-info-card-content p {
+    line-height: 1.5;
+    padding-bottom: 8px;
```

Comment:
Ditto: Use relative units. Here and below, instead of px, let's use `em` or `rem`.